src/orion/storage/legacy.py

```diff
@@ -196,39 +195,25 @@ def register_lie(self, trial):
         """See :func:`orion.storage.base.BaseStorageProtocol.register_lie`"""
         return self._db.write("lying_trials", trial.to_dict())
 
-    def retrieve_result(self, trial, results_file=None, **kwargs):
-        """Parse the results file that was generated by the trial process.
+    def retrieve_result(self, trial, **kwargs):
```

Comment:
seems there is almost no need of this `method`?

Reply:
Almost. Track may use it to update the results of the trial by fetching its db.